src/google/protobuf/compiler/js/js_generator.cc

```diff
@@ -2349,6 +2341,20 @@ void Generator::GenerateClassField(const GeneratorOptions& options,
         "clearedvalue", (field->is_repeated() ? "[]" : "undefined"),
         "returnvalue", JSReturnClause(field));
 
+    printer->Print(
```

Comment:
If there is a clean way, let's try to avoid duplicating this code. At the least, factor it into a shared function.